src/tests/ftest/pool/management_race.py

```diff
@@ -38,20 +38,17 @@ def del_recreate_query_and_list_pools(
         label = pool_label_prefix + str(pool_num)
         for test_num in range(test_loop):
             if del_recreate:
-                destroy_result = self.get_dmg_command().pool_destroy(
-                    pool=self.pool[pool_num].label.value, force=1)
+                self.pool[pool_num].destroy()
                 self.log.info(
-                    "--(%d.2.%d.%d)Pool %s deleted.\n",
-                    thread_num, pool_num, test_num, label)
-                new_pool = self.get_pool()
-                new_pool.set_property(prop_name="label", prop_value=label)
+                    "--(%d.2.%d.%d)Pool %s deleted.\n", thread_num, pool_num, test_num, label)
+                new_pool = self.get_pool(create=False)
```

Comment:
```suggestion
                self.pool[pool_num].create()
```
Like @daltonbohning said, line 44-47 can be replaced with this.